hostlists-builder/index.js

```diff
@@ -253,6 +288,25 @@ async function build(filtersDir, tagsDir, localesDir, assetsDir) {
     filtersMetadataDev.push(filterMetadata);
   }
 
+  // Build Mastodon dynamic server list
+  let servicesJson = JSON.parse(readFile(path.join(assetsDir, SERVICES_FILE)));
+
+  const result = (await mastodonServerList());
+  const mastodonRules = result.map((element) => {
+    return `||${element.domain}^`
+  });
+
```

Comment:
We need additional checks for `mastodonRules`:
1. Not less than N elements (how many is okay)?
2. Not more than X elements (how many is a good upper limit? 100?)

Reply:
100 seems like a fair limit. 

Just as an additional check, you want to sort them by total user count ? So that only the top 100 servers are blocked?